modules/nf-core/sigprofiler/extractor/main.nf

```diff
@@ -0,0 +1,56 @@
+process SIGPROFILER_EXTRACTOR {
+    tag "$meta.id"
+    label 'process_single'
+
+    // TODO nf-core: List required Conda package(s). Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
+    conda "${moduleDir}/environment.yml"
+    container "sigprofilerextractor"
+
+    input:
+    tuple val(meta), path(vcf)
+    val(genome)
+
+    output:
+    tuple val(meta), path("results"), emit: results
```

Comment:
If there are not too many output files, consider specific output channels for them (e.g. `tuple val(meta), path("*.bam"), emit: bam`)